src/test/java/uk/nhs/hee/trainee/details/service/ProgrammeMembershipServiceTest.java

```diff
@@ -1368,14 +1369,16 @@ void rollout2024ShouldBeFalseIfPmHasWrongType(String pmType) {
   }
 
   @ParameterizedTest
-  @MethodSource("listLoPilot2024AllProgrammes")
+  @MethodSource("listLoRollout2024")
```

Comment:
This and subsequent changes are just a bit of a tidy-up to use the actual list of rollout LOs not the pilot all-programme ones.